docs/source/quickstart.rst

```diff
@@ -64,14 +90,19 @@ Run the command below to download the results
 
 .. code-block:: bash
 
-    make get_results
+    gsutil -qm cp ${YOUR_RESULTS_BUCKET}/*.out.gz .
 
-If you are working at an NCBI workstation, you can optionally run the command
-below to perform basic sanity checks on the result files.
+If you are working at an NCBI workstation, after downloading the results you
+can optionally run the command below to perform basic sanity checks on the
+result files.
 
 .. code-block:: bash
 
-    make test_asn_results
+    find . -name "batch*.out.gz" -type f -print0 | \
+        xargs -0 -P8 -I{} -t gzip -t {}
+    find . -name "batch*.out.gz" -type f -print0 | \
+        xargs -0 -P8 -I{} -t bash -c "zcat {} |
+        datatool -m /netopt/ncbi_tools64/c++.metastable/src/objects/blast/blast.asn -M /am/ncbiapdata/asn/asn.all -v - -e /dev/null"
```

Comment:
Does anyone outside of our group use BLAST archive format?

Reply:
I don't know that. The benefit of using this format is that one can reformat in any other format without having to re-run BLAST. I added a link to blast_formatter in the documentation to highlight this.